### PR TITLE
merge lexy-matchpage to main

### PR DESCRIPTION
**Pull request by alexamaingard**

### CODE REVIEW

File: src/client/components/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const Header = () => {
+    return (
+        <>
+        </>
+    )
+}
+
+export default Header
```
Comment (Can-Tina) [inline src/client/components/Header.jsx:10]: This'll be useful, thanks

File: src/client/styles/match.css [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+.tag {
+    background-color: rgb(218, 218, 218);
+    border-radius: 5px;
+    height: fit-content;
+    width: fit-content;
+    padding: 3px 5px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
```
Comment (Can-Tina) [inline src/client/styles/match.css:153]: Love the design

**Review by Can-Tina — APPROVED** [review]

File: src/client/styles/match.css (hunk 2 above)
Comment (TheunisK) [inline src/client/styles/match.css:153]: Really cool, Lexy

**Review by TheunisK — APPROVED** [review]

**Review by hannah68 — APPROVED** [review]

**Review by glowkeeper — APPROVED** [review]
Approved :)

**Pull request merged**